VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/monitoring-graph/monitoring-graph.component.scss

```diff
@@ -60,3 +60,79 @@
   width: 100%;
   margin-top: 30px;
 }
+
+.box {
+  background: #fff;
```

Comment:
govuk colour

Reply:
updated to govuk-colour for all the colours